CRM/Utils/Date.php

```diff
@@ -873,8 +873,11 @@ public static function getRange($startDate, $endDate) {
    */
   public static function getFromTo($relative, $from, $to, $fromTime = NULL, $toTime = '235959') {
     if ($relative) {
-      list($term, $unit) = explode('.', $relative, 2);
-      $dateRange = self::relativeToAbsolute($term, $unit);
+      $dateRange = CRM_Utils_Hook::relativeDate($relative);
```

Comment:
@totten @colemanw I feel like in the past Lobo told me off for using the return value from a hook (& said it should just be about modifying arrays passed by reference) - do you know if there is a reason not to return a value like this?